README.md

```diff
@@ -12,16 +12,73 @@ Manage GitLab configuration as code to make it easily manageable and reproducibl
 ## Usage
 
 
-### Use locally
+### GitLab Configuration
 
 
-### Use in CI/CD pipeline
+### Client configuration
+
+
+### Using Docker image
 
 
 ### Examples
 
 
-## Development
+## Contributing
```

Comment:
Consider to move this to the `CONTRIBUTING` file.

Reply:
Done